Automattic/WP-Job-Manager
# Disable special field validation on draft save

This disables special field validation on draft save. It uses a tiny bit of hackery, but other methods to determine the source of the submit trigger seemed even more hackery-ish.

These features were built independently (#1732 and #1746 ) and need this to make them work together.

#### Changes proposed in this Pull Request:

* Disable validation of job category and job description fields on draft save.

#### Testing instructions:

* Verify you can save a draft without these required fields set.
* Verify normal submission flow _does_ get blocked if these required fields aren't set.

For additional testing, see the original PRs.
